### ISSUE TITLE
`vagga _clean --everything` is not implemented

### ISSUE THREAD

**Issue by sashasimkin** [tailhook/vagga#88]
I believe you know about the issue, but still let it be here. :)
```
$ vagga _clean --everything
thread '<main>' panicked at 'not yet implemented', src/wrapper/clean.rs:80
```

**Comment by tailhook** [comment]
Yeah, it's okay that we now have an issue.

You can achieve the same with:
```
rm -rf .vagga
```
Or if that gives permission error:
```
sudo rm -rf .vagga
```

**Comment by sashasimkin** [comment]
Sure, already did. Thanks!

**Comment by tailhook** [comment]
Fixed in #277
